Classes/item.rb

```diff
@@ -0,0 +1,32 @@
+require 'date'
+
+class Item
+  attr_accessor :publish_date, :archived
+  attr_reader :id, :genre
+
+  def initialize(publish_date)
+    @id = rand(1..1000)
+    @publish_date = publish_date
+    @archived = false
+  end
+
+  def genre=(genre)
+    @genre = genre
+    genre.items.push(self) unless genre.items.include?(self)
+  end
+
+  def author=(author)
+    @author = author
+    author.items << self unless author.items.include?(self)
+  end
+
+  def move_to_archive
+    @archived = can_be_archived?
+  end
+
+  private
+
+  def can_be_archived?
+    Date.today - Date.parse(@publish_date) > 10 * 365
+  end
+end
```

Comment:
- [ ] Kindly add a `label` much like how the `author` and `genre` is added. As per [the given UML diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png).

Reply:
@ShadyShawkat 
thanks for your good and valuable feedback, as you mention the Label is added.